src/Mir/Overrides.hs

```diff
@@ -177,4 +205,20 @@ bindFn fn cfg =
                        let reason = AssumptionReason loc $ "Assumption \n\t" <> src <> "\nfrom " <> locStr
                        liftIO $ addAssumption s (LabeledPred (regValue c) reason)
                        return ()
+               , override "crucible::array::symbolic" (Empty :> strrepr) (UsizeArrayRepr (BaseBVRepr (knownNat @8))) $ do
+                    RegMap (Empty :> str) <- getOverrideArgs
+                    let sym = (undefined :: sym)
+                    x <- maybe (fail "not a constant string") pure (getString sym (regValue str))
```

Comment:
Same comment about error message here.

Reply:
Oh, should this whole `do` block be a call to `array_symbolic` from above?

Reply:
Or could it at least share more with that function?

Reply:
This case should be removed, actually.  It can't ever fire now that `array::symbolic` is polymorphic - crux-mir will only see individual monomorphized instances, which have names like `crucible::array::symbolic::_inst11223344...`, so this override won't match.  The actual implementation of `crucible::array::symbolic` is [here](https://github.com/GaloisInc/mir-verifier/pull/14/files/6e5a5adf02cd00c4e285f3a9d23b8a7071a0d14c#diff-dca1e5fea22e7d9c379ccb863aaa6a3dR103-R109), and it does indeed call `array_symbolic`.